src/w3c/templates/show-people.js

```diff
@@ -2,55 +2,54 @@ export default (conf, name, items = []) => {
   const html = hyperHTML;
   // stuff to handle RDFa
   const attr = Object.freeze(conf.doRDFa ? {
-    ...name === "Editor" ? {
-      rp1: "rdf:first",
-      rp2: "foaf:Person"
-    } : name === "Author" ? {
-      rp1: "dc:contributor",
-      rp2: "foaf:Person"
-    } : {},
-    rn: "foaf:name",
+    re: (name === "Editor" ? "schema:editor" : "schema:contributor"),
+    rp: "schema:Person",
+    rn: "schema:name",
     rm: "foaf:mbox",
-    rwu: "foaf:workplaceHomepage",
-    rpu: "foaf:homepage",
-    propSeeAlso: "rdfs:seeAlso"
+    ru: "schema:url"
   } : {});
-  const bns = [];
-  if (conf.doRDFa && name === "Editor") {
-    bns.push(...Array.from({ length: items.length }, (_, i) => `_:editor${i}`));
-    bns[items.length] = "rdf:nil";
-  }
   const results = [];
   for (let i = 0; i < items.length; i++) {
     results.push(getItem(items[i], i));
   }
   return results;
 
   function getItem(p, i) {
-    const bn = bns[i];
-    const re = (bn && !i) ? "bibo:editor" : null;
     const editorid = p.w3cid ? parseInt(p.w3cid, 10): null;
     const dd = html`<dd class='p-author h-card vcard'
-      property='${re}' resource='${bn}' data-editor-id='${editorid}'></dd>`;
+      data-editor-id='${editorid}'></dd>`;
     const span = conf.doRDFa ?
-      html`<span property='${attr.rp1}' typeof='${attr.rp2}'></span>` :
+      html`<span property='${attr.re}' typeof='${attr.rp}'></span>` :
       document.createDocumentFragment();
     const contents = [];
     if (p.url) {
       if (conf.doRDFa) {
-        contents.push(html`<meta property='${attr.rn}' content='${p.name}' />`);
+        contents.push(html`<meta property='${attr.rn}' content='${p.name}'>`);
       }
       contents.push(html`<a class='u-url url p-name fn'
-        property='${attr.rpu}' href='${p.url}'>${p.name}</a>`);
+        property='${attr.ru}' href='${p.url}'>${p.name}</a>`);
     } else {
       contents.push(html`<span property='${attr.rn}' class='p-name fn'>${p.name}</span>`);
     }
     if (p.company) {
       if (p.companyURL) {
-        contents.push(html`, <a property='${attr.rwu}'
-          class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>`);
+        if (conf.doRDFa) {
+          contents.push(html`, <span property='schema:worksFor' typeof='schema:Organization'>
+            <meta property='${attr.rn}' content='${p.company}'>
+            <a property='${attr.ru}'
+            class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>
+            </span>`);
+        } else {
+          contents.push(html`, <a class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>`);
```

Comment:
We can remove an else branch here as we use `attr`.

Reply:
I don't think this can be cleaned up any further, without leaving an empty span surrounding the anchor.

Reply:
@saschanaz, if you can see a more optimal solution, maybe you can do the changes on-top of this PR? I can live with what is currently here, as there is a high probability we are going to remove all this.